### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by SamoylenkVi**

### CODE REVIEW

File: project/src/components/DetailsNavigation/index.tsx [hunk 1]
```diff
@@ -9,20 +10,33 @@ type DetailsNavigationProps = {
   film: Film;
 }
 
+
 export const DetailsNavigation = ({ film }:DetailsNavigationProps) => {
-  const [currentButton, setCurrentButton] = useState<NavigationButtonsType>(NAVIGATION_BUTTONS.DETAILS);
+
+  const {id} = useParams();
```
Comment (Pavel-Liteiniy) [inline project/src/components/DetailsNavigation/index.tsx]:
```suggestion
  const {id, currentInformation} = useParams();
```

File: project/src/components/DetailsNavigation/index.tsx [hunk 2]
```diff
@@ -9,20 +10,33 @@ type DetailsNavigationProps = {
...
-    if (buttonName) {
+    if (buttonName && id) {
       if (
         Object
           .values(NAVIGATION_BUTTONS)
           .includes(buttonName as NavigationButtonsType)
       ) {
         setCurrentButton(buttonName as NavigationButtonsType);
+
+        const path = generatePath('/films/:id/:currentInformation', {
```
Comment (Pavel-Liteiniy) [inline project/src/components/DetailsNavigation/index.tsx]: let's store it in global constants `/films/:id/:currentInformation`

File: project/src/components/FilmCard/index.tsx [hunk 3]
```diff
@@ -31,30 +31,37 @@ export const FilmCard = ({
...
+    return () => clearTimeout(interval);
+  }, [isVideo]);
+
   return (
     <article onMouseEnter={playVideo} onMouseLeave={stopVideo} className="small-film-card catalog__films-card">
       <div className="small-film-card__image">
         {(isVideo && !isVideoError)
-          ? <VideoPlayer src={previewVideoLink} ref={videoRef} />
+          ? <VideoPlayer src={previewVideoLink} ref={videoRef} poster={previewImage}/>
           : <img src={previewImage} alt={altText} width="280" height="175" />}
```
Comment (Pavel-Liteiniy) [inline project/src/components/FilmCard/index.tsx:60]: Do you need this image if you use poster attr?

File: project/src/mocks/films.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { Film } from '../types/film';
 
 export const films: Film[] = [
   {
-    id: nanoid(),
+    id: '1',
```
Comment (Pavel-Liteiniy) [inline project/src/mocks/films.ts]: let's made static ids for all films

**Review by Pavel-Liteiniy — CHANGES_REQUESTED** [review]

File: project/src/components/DetailsNavigation/index.tsx (hunk 1 above)
Comment (SamoylenkVi) [reply project/src/components/DetailsNavigation/index.tsx]: fixed

File: project/src/components/DetailsNavigation/index.tsx (hunk 2 above)
Comment (SamoylenkVi) [reply project/src/components/DetailsNavigation/index.tsx]: fixed

File: project/src/mocks/films.ts (hunk 4 above)
Comment (SamoylenkVi) [reply project/src/mocks/films.ts]: fixed

**Comment by Pavel-Liteiniy** [comment]
Approved! Well done! 👍

**Pull request merged**